### PR TITLE
fix: skip Accept header parsing in case no Accept headers presented

### LINKED ISSUE #520: Bug: Accept.values() fails with java.lang.IllegalStateException: Invalid mime type format is no Accept header specified
Accept.values() fails with exception when no Accept header was provided

`java.lang.IllegalStateException: Invalid mime type format
	at wtf.g4s8.mime.MimeTypeOfString.typeMatcher(MimeTypeOfString.java:113)
	at wtf.g4s8.mime.MimeTypeOfString.type(MimeTypeOfString.java:69)
	at wtf.g4s8.mime.MimeType$1.test(MimeType.java:73)
	at wtf.g4s8.mime.MimeType$1.test(MimeType.java:68)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
	at java.base/java.util.stream.SortedOps$SizedRefSortingSink.end(SortedOps.java:357)
	at java.base/java.util.stream.Sink$ChainedReference.end(Sink.java:258)
	at java.base/java.util.stream.Sink$ChainedReference.end(Sink.java:258)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:510)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at wtf.g4s8.mime.MimeType.parse(MimeType.java:91)
	at com.artipie.http.headers.Accept.values(Accept.java:54)`

### PR DESCRIPTION
**Pull request by sunmeplz**
closes #520

### CODE REVIEW

File: src/main/java/com/artipie/http/headers/Accept.java [hunk 1]
```diff
@@ -40,10 +42,16 @@ public Accept(final Iterable<Map.Entry<String, String>> headers) {
      * Parses `Accept` header values, sorts them according to weight and returns in
      * corresponding order.
      * @return Set or the values
+     * @checkstyle ReturnCountCheck (11 lines)
      */
+    @SuppressWarnings("PMD.OnlyOneReturn")
```
Comment (ChGen) [inline src/main/java/com/artipie/http/headers/Accept.java:47]: Could we avoid `@SuppressWarnings`  by having single return?

**Review by ChGen — COMMENTED** [review]
Could

**Review by dgarus — APPROVED** [review]
@sunmeplz Thanks!
Looks good to me.

**Review by andpopov — APPROVED** [review]
@sunmeplz Thanks!

**Pull request merged**